src/App.js

```diff
@@ -20,7 +19,30 @@ function App() {
   return (
     <div className="App">
       <header className="App-header sticky" id="header">
+        <ul>
+          <li>
+            <a>Period Friendly Bristol</a>
+          </li>
+          {/* <li>
+            <img src={logo} className="App-logo mg-left-2" alt="logo" />
+          </li> */}
+        </ul>
       </header>
+      <div className="container-middle">
```

Comment:
Is there anyway this about page can updated locally with what is current on `master`?

Reply:
Nvm this is outdated.